app/forms/project_form.rb

```diff
@@ -66,4 +67,15 @@ def invite_to_github_team
     username = user.nickname
     github.invite_to_team(team_id, username)
   end
+
+  def create_new_repo_on_organization
```

Comment:
Assignment Branch Condition size for create_new_repo_on_organization is too high. [16.12/15]